Assets/HoloToolkit/Utilities/Scripts/FixedAngularSize.cs

```diff
@@ -38,10 +47,25 @@ void Start()
             }
         }
 
+        public void SetSizeRatio(float ratio)
+        {
+            if (ratio == 0)
```

Comment:
That sounds good, I just moved the logic from Start into the SetSizeRatio function so we get all the protection without duplicating code.